gym_cap/envs/create_map.py

```diff
@@ -107,3 +151,31 @@ def populate_map(new_map, code_where, code_what):
                 break
         new_map[lx,ly] = code_what
         return new_map
+    
+    @staticmethod
+    def depopulate_map(all_agent_map, code_where, code_what):
+        """
+        Function
+            Replaces "code_what" with a random location of "code_where" at "all_agent_map"
+
+        Parameters
+        ----------
+        new_map     : 2d numpy array
+            Map of the environment
+        code_where  : int
+            Code of the territory that is being populated
+        code_what   : int
+            Value assigned to the random location of the map
+        """
+        dimx, dimy = all_agent_map.shape
+        if code_what in all_agent_map:
+            while True:
+                lx = np.random.randint(0, dimx)
+                ly = np.random.randint(0, dimy)
+                if all_agent_map[lx, ly] == code_what:
+                    break
+            all_agent_map[lx, ly] = code_where
```

Comment:
The code is repeated.

Reply:
In createmap.py, depopulate_map uses the same variables as populate_map but it’s function is opposite to that of populate_map

Reply:
Why not use ```populate_map(all_agent_map, code_what, code_where)```? It is 'de'populating, right?